bindata/routes/downloads-route.yaml

```diff
@@ -1,12 +1,10 @@
+# Default 'downloads' route manifest.
+# The 'downloads' route will be pointing to the 'downloads' service. 
 apiVersion: route.openshift.io/v1
 kind: Route
 metadata:
   namespace: openshift-console
   name: downloads
-  annotations:
-    include.release.openshift.io/ibm-cloud-managed: "true"
-    include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
```

Comment:
Why aren't routes not included in these releases?

Reply:
I had not noticed that this file was moved from the manifests dir. CVO is no longer managing this resource.